kwok/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Start from a base image
+FROM registry.access.redhat.com/ubi8:latest
+
+# Update packages
+RUN yum update -y && yum clean all
+
+RUN yum install -y bash
+
+# Install curl and other dependencies
+RUN yum install -y curl tar && yum clean all
+
+# Install Kwok & Kwokctl
+RUN KWOK_VERSION=$(curl -s https://api.github.com/repos/kubernetes-sigs/kwok/releases/latest | grep 'tag_name' | cut -d\" -f4) && \
```

Comment:
`curl`ing during a build is a bad idea - it will not work in RHTAP. 

Either:
1. Use a base image that already contains kwok
2. Use the golang tools to vendor the kwok source code as a dependency (I assume its written in Go) and build it during the container build.

Reply:
I tried the second approach there were some dependencies that is unresolved. 
[root@5d851c9cbae6 /]# kwok
kwok: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by kwok)
kwok: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by kwok)
[root@5d851c9cbae6 /]# kwokctl
kwokctl: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by kwokctl)
kwokctl: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by kwokctl)

Reply:
it seems you need to install the devel version of glibc to build kwok.

did you use the go toolset image to build?

Reply:
Actually, looking again at the erorr - it seems the library is missing at runtime - are you using the same RHEL/UBI version to build and run?

Reply:
go-toolset latest id 1.19 and kwok requires 1.20 to install via go mod. so did a workaround to install via debian

Reply:
so that is causing your problem because Debian's libc is very different from Red Hat's - you need to find a way to only use Red Hat stuff - perhaps we can sun some command in the toolset to update go there somehow

Reply:
updated to use v0.2.0

Reply:
not sure I follow, are you referring to the kwok version?

Reply:
yep